package.json

```diff
@@ -49,7 +49,8 @@
     "ts-node": "10.9.1",
     "tslint": "5.5.0",
     "typescript": "^4.7.4",
-    "vsce": "2.15.0"
+    "vsce": "2.15.0",
+    "ovsx": "latest"
```

Comment:
`npx ovsx publish......` command uses latest ovsx by default. So not pinning it to any version but latest.